src/Filemanager.tsx

```diff
@@ -68,42 +83,42 @@ function Filemanager() {
   };
 
   const handleContextMenu = (
-    { label, nodeData }: ITreeNode<api.File>,
+    node: ITreeNode<api.File>,
     nodePath: number[],
     e: React.MouseEvent<HTMLElement>
   ) => {
     e.preventDefault();
-    if (nodeData === undefined) {
+    if (node === undefined) {
       throw new Error("assertion: nodeData must not be undefined");
     }
```

Comment:
I think you can remove those 3 lines. They make no sense at all. node is a ITreeNode<api.File>, it will NEVER be undefined